### PR TITLE
Added Contributing.md

### PR DESCRIPTION
**Pull request by ira-mane**
Hello @Harshal141 . Please have a look on the contributing.md file and let me know any issue.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+1. Fork the repository to your GitHub account.
+   1. Click the fork button on the repository.
+   2. Click Create Fork button below down.
+   3. Copy the ssh clone url.
+2. Clone your forked repository to your local machine.
+   1. Cloning the forking repository using git clone command.
+3. Create a new branch for your changes.
+4. Make your changes and commit them with clear, concise commit messages.
+5. Push your branch to your forked repository.
+6. Open a pull request from your branch to our repository's main branch.
```
Comment (Harshal141) [inline CONTRIBUTING.md:28]: `master` branch not `main` branch.
and also add ` back ticks to mention branches.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Development Environment setup
+
+If you plan to contribute code, follow these steps to set up your development environment:
+
+1. Clone the repository.
+2. Start the development of the project.
+
+Thank you for your interest in contributing to ResumeWorded ! We welcome contributions from the community and appreciate your help in making our project better.
+
```
Comment (Harshal141) [inline CONTRIBUTING.md:40]: Can we add a section for `local deployment`
- Steps on how contributers can setup the repo and run the project locally.
- A brief about the folder structure of the project.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+### Submitting an issue :-
+
+If you feel ,there's any bug or feature improvement Please raise an issue.
+We are here to improve 🙌
+
+### Submitting Pull Request :-
+If you'd like to contribute code or documentation changes, you can submit a pull request (PR) to our repository. Here's how:
+
+1. Fork the repository to your GitHub account.
```
Comment (Harshal141) [inline CONTRIBUTING.md:19]: It would be great if we have picture showing these steps.
many people are new to oss so that would be helpful

**Review by Harshal141 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (ira-mane) [reply CONTRIBUTING.md:19]: Okay sure ! I will add these , If I have any doubt I will let you know .

File: CONTRIBUTING.md (hunk 2 above)
Comment (ira-mane) [reply CONTRIBUTING.md:40]: > Can we add a section for `local deployment`
> 
> * Steps on how contributers can setup the repo and run the project locally.
> * A brief about the folder structure of the project.

This part is already there in the src folder readme file

Comment (Harshal141) [reply CONTRIBUTING.md:57]: ok just add about the local exectution steps.
After updating the changes you can request re-review.

**Comment by ira-mane** [comment]
Please review the PR

**Review by Harshal141 — APPROVED** [review]

**Pull request merged**